link2aws.js

```diff
@@ -1084,4 +1084,4 @@ if (typeof (require) !== 'undefined' && require.main === module) {
     }
 }
 
-exports.ARN = ARN;
\ No newline at end of file
+exports.ARN = ARN;
```

Comment:
Could you undo the unnecessary whitespace change here?

Other than that, looks good to me. And thanks, I appreciate it!

(Git tip: interactive add (`git add -p`) is quite nice to avoid staging accidental changes.)

Reply:
Apologies, my VSCode has the setting `Insert Final Newline` enabled. Fixed.